src/AutoGolem.cs

```diff
@@ -84,6 +85,28 @@ private void OnSettingsToggle()
                     isTown = GameController.Area.CurrentArea.IsTown;
 
                     keyboard = new KeyboardHelper(GameController);
+                    targetNumber = 0;
+
+                    if (Settings.ChaosGolem.Value)
+                    {
+                        targetNumber += Settings.ChaosGolemMax.Value;
+                    }
+                    if (Settings.FireGolem.Value)
+                    {
+                        targetNumber += Settings.FireGolemMax.Value;
+                    }
+                    if (Settings.IceGolem.Value)
+                    {
+                        targetNumber += Settings.IceGolemMax.Value;
+                    }
+                    if (Settings.LightningGolem.Value)
+                    {
+                        targetNumber += Settings.LightningGolemMax.Value;
+                    }
+                    if (Settings.StoneGolem.Value)
+                    {
+                        targetNumber += Settings.StoneGolemMax.Value;
+                    }
```

Comment:
targetNumber will not be recalculated if the player changes it after the plugin is initialized